doc/whats_new.rst

```diff
@@ -135,6 +137,7 @@ People who contributed to this release (in alphabetical order):
 - `Blake Caldwell`_
 - `Christopher Bailey`_
 - `Carmen Kohl`_
+- `Kenneth Loi`_
```

Comment:
```suggestion
```

this is contributors for 0.1 release. We'll add you to the contributor list for 0.2 :)